webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+* **`ensembles`:** Which ensembles in `shared_settings` to visualize. The lack of `response_file` \
+                implies that the input data should be time series data from simulation `.UNSMRY` \
+                files, read using `fmu-ensemble`.
+* **`column_keys`:** (Optional) list of simulation vectors to include as responses when reading \
+                from UNSMRY-files in the defined ensembles (default is all vectors). * can be \
+                used as wild card.
+* **`sampling`:** (Optional) sampling frequency when reading simulation data directly from \
+               `.UNSMRY`-files (default is monthly).
+
+?> The `UNSMRY` input method implies that the "DATE" vector will be used as a filter \
+   of type `single` (as defined below under `response_filters`).
+
+
+**Common settings for all input options**
+
+All of these are optional, some have defaults seen in the code snippet below.
+
+* **`response_filters`:** Optional dictionary of responses (columns in csv file or simulation \
+                       vectors) that can be used as row filtering before aggregation. \
+                       Valid options:
+    * `single`: Dropdown with single selection.
+    * `multi`: Dropdown with multiple selection.
+    * `range`: Slider with range selection.
+* **`response_ignore`:** List of response (columns in csv or simulation vectors) to ignore \
+                      (cannot use with response_include).
+* **`response_include`:** List of response (columns in csv or simulation vectors) to include \
+                       (cannot use with response_ignore).
+* **`parameter_ignore`:** List of parameters (columns in csv or simulation vectors) to ignore
+* **`aggregation`:** How to aggregate responses per realization. Either `sum` or `mean`.
+
+
+**Note**: Regression models break down when there are duplicate or highly correlated parameters. \
+Please make sure to properly filter your inputs or the model will give answers that are misleading.
+
+---
+
+?> Non-numerical (string-based) input parameters and responses are removed.
+
+?> The responses will be aggregated per realization; meaning that if your filters do not reduce \
+the response to a single value per realization in your data, the values will be aggregated \
+accoording to your defined `aggregation`. If e.g. the response is a form of volume, \
+and the filters are regions (or other subdivisions of the total volume), then `sum` would \
+be a natural aggregation. If on the other hand the response is the pressures in the \
+same volume, aggregation as `mean` over the subdivisions of the same volume \
+would make more sense (though the pressures in this case would not be volume weighted means, \
+and the aggregation would therefore likely be imprecise).
+
+!> Regression models break down when there are **duplicate or highly correlated parameters**. \
+Please make sure to properly filter your inputs or the model will give answers that are misleading.
```

Comment:
Something like this could be added in the description as well as mentioned above